gae/js/app/views/sitecaptain.js

```diff
@@ -71,12 +71,12 @@ define(
             addCaptain: function(e) {
                 e.preventDefault();
                 var self = this;
-                var choices = this.form.fields.models.find(function(model){return model.attributes.name == 'captain';});
-                var captain_id = this.model.attributes.captain;
-                this.captain  = _.findWhere(choices.changed.options, {value: captain_id});
+                var choices = this.form.fields.models.find(function(model){return model.get('name') == 'captain';});
+                var captain_id = this.model.get('captain');
+                var captain  = _.findWhere(choices.get('options'), {value: captain_id});
 
```

Comment:
Do I really need to declare var captain_id?
I declared it for readability but don't like it anymore.
What do you think about passing {value: this.model.get('captain')} instead?